app/decorators/order_decorator.rb

```diff
@@ -0,0 +1,15 @@
+class OrderDecorator < Draper::Decorator
+  delegate_all
+
+  def status_string
+    case object.status
+    when 'in_queue' then I18n.t('orders.waiting')
```

Comment:
Add status key to locale file. For examle:
```yml
orders:
  in_queue: text...
```
Then you can update method like this:
```ruby
def status_string
  I18n.t("orders.#{object.status}")
end
```
Using `case` is bad practice.